<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QvisConvertDialog</class>
 <widget class="QDialog" name="QvisConvertDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>312</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Convert</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <selectedon>:/Src/FluoRender/UI/icons/Tools/Convert.png</selectedon>
   </iconset>
  </property>
  <layout class="QVBoxLayout" name="MainLayout">
   <item>
    <widget class="QGroupBox" name="ConvertGroupBox">
     <property name="title">
      <string>Volume to Polygon Mesh</string>
     </property>
     <layout class="QVBoxLayout" name="ConvertLayout">
      <item>
       <layout class="QGridLayout" name="ConvertValuesLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="ThresholdLabel">
          <property name="text">
           <string>Threshold</string>
          </property>
          <property name="buddy">
           <cstring>ThresholdSpinBox</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSlider" name="ThresholdSlider">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="value">
           <number>30</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QDoubleSpinBox" name="ThresholdSpinBox">
          <property name="minimum">
           <double>0.010000000000000</double>
          </property>
          <property name="maximum">
           <double>0.990000000000000</double>
          </property>
          <property name="value">
           <double>0.300000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="DownSampleXYLabel">
          <property name="text">
           <string>Down sample XY</string>
          </property>
          <property name="buddy">
           <cstring>DownSampleXYSpinBox</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSlider" name="DownSampleXYSlider">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>10</number>
          </property>
          <property name="value">
           <number>2</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QSpinBox" name="DownSampleXYSpinBox">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>10</number>
          </property>
          <property name="value">
           <number>2</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="DownSampleZLabel">
          <property name="text">
           <string>Down sample Z</string>
          </property>
          <property name="buddy">
           <cstring>DownSampleZSpinBox</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSlider" name="DownSampleZSlider">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>10</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QSpinBox" name="DownSampleZSpinBox">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>10</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="ConvertOptionsLayout">
        <item row="0" column="0">
         <widget class="QCheckBox" name="UseTransferFunctionCheckBox">
          <property name="text">
           <string>Use transfer function</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="SelectedOnlyCheckBox">
          <property name="text">
           <string>Selected only</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QCheckBox" name="WeldVerticesCheckBox">
          <property name="text">
           <string>Weld vertices</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="ButtonLayout">
     <item>
      <widget class="QPushButton" name="ConvertButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Convert</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="ButtonSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="OutputGroupBox">
     <property name="title">
      <string>Output</string>
     </property>
     <layout class="QVBoxLayout" name="OutputLayout">
      <item>
       <widget class="QTextEdit" name="OutputText">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>ThresholdSlider</tabstop>
  <tabstop>ThresholdSpinBox</tabstop>
  <tabstop>DownSampleXYSlider</tabstop>
  <tabstop>DownSampleXYSpinBox</tabstop>
  <tabstop>DownSampleZSlider</tabstop>
  <tabstop>DownSampleZSpinBox</tabstop>
  <tabstop>UseTransferFunctionCheckBox</tabstop>
  <tabstop>WeldVerticesCheckBox</tabstop>
  <tabstop>SelectedOnlyCheckBox</tabstop>
  <tabstop>ConvertButton</tabstop>
  <tabstop>OutputText</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>ThresholdSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>QvisConvertDialog</receiver>
   <slot>ThresholdSliderChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>180</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>-7</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ThresholdSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>QvisConvertDialog</receiver>
   <slot>ThresholdValueChanged(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>250</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>307</x>
     <y>9</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DownSampleXYSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>QvisConvertDialog</receiver>
   <slot>DownSampleXYSliderChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>192</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>307</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DownSampleXYSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>QvisConvertDialog</receiver>
   <slot>DownSampleXYValueChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>256</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>308</x>
     <y>100</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DownSampleZSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>QvisConvertDialog</receiver>
   <slot>DownSampleZSliderChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>167</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>308</x>
     <y>136</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DownSampleZSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>QvisConvertDialog</receiver>
   <slot>DownSampleZValueChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>256</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel">
     <x>307</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>UseTransferFunctionCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>QvisConvertDialog</receiver>
   <slot>UseTransferFunctionToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>128</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel">
     <x>306</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>WeldVerticesCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>QvisConvertDialog</receiver>
   <slot>WeldVerticesToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>55</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>7</x>
     <y>243</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SelectedOnlyCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>QvisConvertDialog</receiver>
   <slot>SelectedOnlyToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>79</x>
     <y>208</y>
    </hint>
    <hint type="destinationlabel">
     <x>6</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ConvertButton</sender>
   <signal>clicked()</signal>
   <receiver>QvisConvertDialog</receiver>
   <slot>ConvertClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>250</x>
     <y>212</y>
    </hint>
    <hint type="destinationlabel">
     <x>306</x>
     <y>215</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>ThresholdSliderChanged(int)</slot>
  <slot>ThresholdValueChanged(double)</slot>
  <slot>DownSampleXYSliderChanged(int)</slot>
  <slot>DownSampleXYValueChanged(int)</slot>
  <slot>DownSampleZSliderChanged(int)</slot>
  <slot>DownSampleZValueChanged(int)</slot>
  <slot>UseTransferFunctionToggled(bool)</slot>
  <slot>WeldVerticesToggled(bool)</slot>
  <slot>SelectedOnlyToggled(bool)</slot>
  <slot>ConvertClicked()</slot>
 </slots>
</ui>
